tests/syntax/test_specifiers.py

```diff
@@ -508,9 +508,9 @@ def test_visible():
     for i in range(30):
         scene = sampleScene(scenario, maxIterations=10)
         ego, base = scene.objects
-        assert ego.position.distanceTo(base.position) <= 10
-        assert ego.position.x >= base.position.x
-        assert ego.position.y >= base.position.y
+        assert ego.position.distanceTo(base.position) <= 10 + math.sqrt(3 * 0.5**2)
+        assert ego.position.x >= base.position.x - math.sqrt(3 * 0.5**2)
+        assert ego.position.y >= base.position.y - math.sqrt(3 * 0.5**2)
```

Comment:
Nitpick: the repetition pains me -- can you just compute the radius once right after the scenario and use it here? Likewise for the other tests.